### DISCUSSION TITLE
Command to integrate toggl report for the day in daily note

### DISCUSSION THREAD

**Discussion by Rhydderch** [mcndt/obsidian-toggl-integration#36] [Feature Requests]
Hi,

Thank you for your plugin.
I was wondering whether it would be possible to add a command that would insert (at the cursor's place) a brief summary of the timed tracked for the day (**using the date of the daily note currently active**).

This would allow me to quickly add a daily summary of time tracked to my daily notes, either at the end of my day, the next morning, or during my weekly review.

It could be as simple as something like this (and maybe with spans to color the name of the project)
```
- 01:30 - <span style="color:red;font-weight:800;">Project Name</span> - Description
- 02:12 - <span style="color:blue;font-weight:800;">Project Name</span> - Description
- 00:23 - <span style="color:green;font-weight:800;">Project Name</span> - Description
```

This would help me stay in Obsidian :)
Let me know if I need to clarify anything and if you think this is possible!

EDIT: I just saw this https://github.com/mcndt/obsidian-toggl-integration/issues/27
This is even more extensive than what I hopped for, so it's great :D

**Comment by mcndt** [comment]
Hi @Rhydderch!

I see you saw the feature that is already in development. Are custom codeblocks a good solution for you?

The development of that feature has been asleep for a bit because I've been very busy with my Master dissertation, but I believe I should have some time to pick it up again soon and push it out somewhere in January.
